### DISCUSSION TITLE
yutto 有继续支持 Python 3.8 的计划吗？

### DISCUSSION THREAD

**Discussion by lesca** [yutto-dev/yutto#19] [General]
为什么不支持3.8了呢。Ubuntu 20.04 LTS 默认就是3.8，而且LTS有5年支持时间。如果换非LTS的版本，就意味着需要经常更新系统版本。
所以还是期待能够支持3.8.

**Comment by SigureMo** [comment]
首先，我不是很明白更新 Python 就一定要更新系统，貌似没有一个系统是只能装一个 Python 吧？至于怎么装，这个搜索引擎一搜就能搜到很多解决方案，甚至你搜索 yutto 都能搜到 Python3.9 的安装方式，比如 <https://blog.vlssu.com/views/deploy-website/yutto.html#%E5%AE%89%E8%A3%85yutto>。

其实吧，原计划是仅支持 3.10 的……这也是 yutto 是被叫做「可爱且**任性**的下载器」的其中一个原因，yutto 不会总是为兼容性而让步，不然这只会让 yutto 步了 bilili 的后尘。

技术上，支持 3.8 应该是没有太大问题的，但写起来没有 3.9 舒服，没错，只是任性而已(๑>؂<๑）

不过 yutto 依然处于 beta 版本，说不定哪天我心血来潮就支持了呢。至于后续计划，我会参考一些反馈建议，如果认为支持 3.8 是有必要的人比较多，我会考虑进行支持的。

**Reply by TommcyOWO** [reply]
我嘗試pip install 但無法正常下載... 3.8 現在還支持嗎

**Reply by SigureMo** [reply]
> 3.8 現在還支持嗎

从未支持过
